R/utils.R

```diff
@@ -101,12 +102,11 @@ is_logical_vector_modif <- function(x, min_length = 1) {
 #'
 #' @export
 rtables_aligns <- function() {
-  c("left", "right", "center")
+  list_valid_aligns()
 }
 
-chk_rtables_align <- function(algn) {
-    if(any(is.na(algn) | !(algn %in% rtables_aligns())))
-        stop("Unsupported text-alignment: ", algn)
+chk_rtables_align <- function(algn) { # xxx duplication with formatters
```

Comment:
`check_aligns` is exported from formatters, lets just refactor to use that directly rather than doing this (as your comment correctly points out) silly duplication.